### ISSUE TITLE
Improvement on closing the send button window

### ISSUE THREAD

**Issue by frtug** [ambujraj/ByteShare#139]
**Is your feature request related to a problem? Please describe.**
currently we only have only the close button to close send button tab, We need to have other ways to close that.
like, User clicks other than main element of close button , it should closes send window 

**Describe the solution you'd like**
Find the alternatives of closing the send button tab 

**Additional context**
[
![Screenshot 2024-04-10 at 16 22 30](https://github.com/ambujraj/ByteShare/assets/32640258/af2da8de-8fad-4fbf-a995-ce3fe474fe35)
](url)

**Comment by ambujraj** [comment]
Hi @frtug. 

This is not an issue. 
Instead, this has been made to be as it is right now just to make sure users don't accidently close the upload dialog resulting in loss of selected files to upload.

**Comment by frtug** [comment]
yes, Its an improvement, we can ask permission if they already have some files selected
